### PR TITLE
Add combination slot type

### PR DESCRIPTION
**Pull request by hskim881028**
### Description

- Add combination slot type(`WaitingReceive`). This type sends `RapidCombination` action and states the state until it is rendered.
 
- Fixed item option tag not showing when the slot type (‘Appraise’)

### How to test

- Make an item

### Related Links

- [워크샵 슬롯에서 즉시제작 후 메일 전송중 상태 프리펩에 추가](https://app.asana.com/0/1133453747809944/1200995510801068/f)

### Required Reviewers

@Namyujeong , @planetarium/9c-dev , @BBeom06 

### Screenshot

test
ezgif com-gif-maker (6)
ezgif com-gif-maker (5)

### CODE REVIEW

**Review by Namyujeong — APPROVED** [review]
✅ When identifying items, the stars disappeared.
✅ Added 'Receiving item through mail' status
✅ Fixed helppopup data > @BBeom06 이미지 너무 이상하면 수정해주셔도 됩니다..

스크린샷 2021-09-17 오전 12 45 11
스크린샷 2021-09-17 오전 12 57 32
스크린샷 2021-09-17 오전 12 57 39
스크린샷 2021-09-17 오전 12 57 47
스크린샷 2021-09-17 오전 12 58 05
스크린샷 2021-09-17 오전 1 38 38

File: nekoyume/Assets/_Scripts/UI/Module/Item/ItemView.cs [hunk 1]
```diff
@@ -185,6 +185,43 @@ public void SetData(TViewModel model, bool isConsumable)
             UpdateView();
         }
 
+        public void SetDataExceptOptionTag(TViewModel model)
```
Comment (Unengine) [inline nekoyume/Assets/_Scripts/UI/Module/Item/ItemView.cs:188]: Does it disable existing option tag enabled before?

Comment (boscohyun) [reply nekoyume/Assets/_Scripts/UI/Module/Item/ItemView.cs:188]: This method has few lines like below differ with `void SetDate(TViewMedel)`.

```
            var tagData = optionTagData.GetOptionTagData(row.Grade);
            Model.HasOptions.Subscribe(hasOptions => SetOptionTag(hasOptions, tagData))
                .AddTo(_disposablesAtSetData);
```

Could this refactor like below?

```
1. public override void SetData(TViewModel) -> public void SetData(TViewModel, bool)

2. public void SetDataExceptOptionTag(TViewModel) -> delete and use above method.
```

Comment (hskim881028) [reply nekoyume/Assets/_Scripts/UI/Module/Item/ItemView.cs:188]: There are cases where hasOptions is true , but the view is disabled. like the screenshot below
image

**Review by boscohyun — APPROVED** [review]

**Pull request merged**